content/how-to/cloud-servers/multifactor-authentication-from-the-cloud-control-panel/index.md

```diff
@@ -89,37 +89,42 @@ To register and verify an SMS device, use the following steps:
 
 2. In the upper-right corner of the Cloud Control Panel, click the **user** menu
    and select **My Profile & Settings**.
+   
+   <img width="383" alt="chrome_kE2DgyCefB" src="https://user-images.githubusercontent.com/96761530/160662203-c5eabb2d-29f5-4d72-aaa0-9109016ffb21.png">
```

Comment:
We should store images within the same folder as article instead of linking to a github user content.